src/WopiValidator.Core/ExceptionHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+
+namespace Microsoft.Office.WopiValidator.Core
+{
+	class ExceptionHelper
+	{
+		private static IResponseData CustomResponseData(string exceptionMessage, string exceptionDetails)
+		{
+			var body = Encoding.UTF8.GetBytes(exceptionDetails);
+			var headers = new Dictionary<string, string> { { Constants.Headers.ValidatorError, exceptionMessage } };
```

Comment:
It's a little hacky to add this to the headers, but I totally get why you're doing it this way.

Reply:
I agree it's a bit hacky. I'll add more comments to the class to explain the rationale. It can be refactored into something more general later if needed.